trin-core/src/portalnet/overlay_service.rs

```diff
@@ -1773,7 +1773,29 @@ where
     }
 
     /// Starts a FindNode query to find nodes with IDs closest to target, starting with given enrs.
-    fn init_find_nodes_query_with_initial_enrs(&mut self, target: &NodeId, enrs: Vec<Enr>) {
+    fn init_find_nodes_query(&mut self, target: &NodeId) {
```

Comment:
docstring got a little stale (no `given enrs` are supplied). Anything else that should be added?